doc/how-to/seed_cas2_data_in_non_production_environments.md

```diff
@@ -0,0 +1,50 @@
+# How to seed CAS2 data in non-production environments
+
+The seeding of `Cas2Application` and `Cas2Assessment` data is mostly done automatically for developers when the API
```

Comment:
I think there might also be value in talking a little about the way seeded data is cleared using these SQL scripts. As in, seeded data is meant to applied afresh on every app restart. This means that seeded data should likely be paired with a relevant `TRUNCATE` found in the `resources/db/migration/<env>.sql`.